src/web/app/components/question-types/question-statistics/mcq-question-statistics.component.ts

```diff
@@ -124,4 +132,31 @@ export class McqQuestionStatisticsComponent
     }
   }
 
+  private sortChoices = (item1: string, item2: string): number => {
+    return this.tableComparatorService.compareLexicographically(item1, item2, SortOrder.ASC);
+  }
+
+  private sortNumbers(item1: number, item2: number): number {
+    return item1 - item2;
+  }
```

Comment:
Should this be in the `TableComparatorService`?

Reply:
You can take a look at my PR #10155 for the use of `TableComparatorService `. Basically all comparison should be delegated to `this.tableComparatorService.compare()`.